geocoding/geocode.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+import googlemaps
+import os
```

Comment:
Prefer `Pathlib` over `os` for cross-os consistency

Reply:
Removed `os`.
Added `Pathlib`